Flaky test_disk_over_web_server

https://s3.amazonaws.com/clickhouse-test-reports/0/4b02d83999808ab06320ba82a4530bb958e6b51f/integration_tests__tsan__[5_6].html

It looks we did not stop ClickHouse when we started `static-files-disk-uploader` and it deleted a file or we just have incorrect data in path.

```
E               Exception: Command ['docker', 'exec', '-u', 'root', 'roottestdiskoverwebserver_node1_1', 'bash', '-c', '/usr/bin/clickhouse static-files-disk-uploader --test-mode --url http://nginx:80/test1 --metadata-path /store/6b3/6b3eb353-292f-499c-9bf3-6fe329a43b89/'] return non-zero code 1: Data path: "/store/6b3/6b3eb353-292f-499c-9bf3-6fe329a43b89", destination path: "http://nginx:80/test1"
E               std::exception. Code: 1001, type: std::__1::__fs::filesystem::filesystem_error, e.what() = filesystem error: in file_size: No such file or directory ["/store/6b3/6b3eb353-292f-499c-9bf3-6fe329a43b89/tmp_merge_all_1_10_2"]
E               Cannot print extra info for Poco::Exception (version 23.6.1.1 (official build))
```

--- a/tests/integration/test_disk_over_web_server/test.py
+++ b/tests/integration/test_disk_over_web_server/test.py
@@ -45,6 +45,8 @@ def cluster():
                 f"CREATE TABLE data{i} (id Int32) ENGINE = MergeTree() ORDER BY id SETTINGS storage_policy = 'def', min_bytes_for_wide_part=1;"
             )
 
+            node.query("SYSTEM STOP MERGES")
+
             for _ in range(10):
                 node.query(
                     f"INSERT INTO data{i} SELECT number FROM numbers(500000 * {i+1})"

